bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditorCssReference.java

```diff
@@ -0,0 +1,27 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+public class SummernoteEditorCssReference extends CssResourceReference {
```

Comment:
Can we use Webjars?
http://search.maven.org/#artifactdetails%7Corg.webjars.bower%7Csummernote%7C0.6.16%7Cjar
I am not sure because of the `.bower` part in the groupId.